build/targets.mk

```diff
@@ -410,15 +472,15 @@ ifeq ($(TARGET),ANDROID)
   LLVM_PREFIX = $(ANDROID_TOOLCHAIN)/bin/
 
   ifeq ($(X86),y)
-    LLVM_TARGET = i686-none-linux-android
+    LLVM_TARGET := i686-linux-android
```

Comment:
Huh?

Reply:
See distinct commit describing adjustments to the LLVM_TARGET for Android builds.

Reply:
"The template can be taken from the prefixed compiler commands in toolchains/llvm/prebuilt/linux-x86_64/bin."  .... but these are commands, not LLVM targets. The LLVM_TARGET variable is not used to build a command, it is only a parameter. This needs further explanation!

Reply:
The mentioned commands a simple shell scripts which invode clang with the correct -target option. I meant the triplet which is the -target parameter in these command shell scripts.
I will amend the commit message.

Reply:
But this commit does three things:
1. change `=` to `:=` (why?)
2. drop `-none` from the LLVM target (why?)
3. append $(ANDROID_NDK_API) to the LLVM target

You explained only 3.
About 2 (dropping `-none`): that may or may not be correct, but you didn't explain it. Look at https://developer.android.com/ndk/guides/standalone_toolchain and you'll see that Google's official NDK documentation tells you to use `-none`, while the shell wrappers don't use it. In any case, if you want to drop `-none`, explain why, and make it a separate commit.